From d4e0b5420ff2af2e790f12d10996f93ec6066b4a Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 1 Jun 2015 11:41:09 +1000
Subject: [PATCH xf86-input-libinput] Fix missing scroll methods default/scroll
 button property

Even if no scroll method is enabled by default, we still want those
properties.

Introduced in 8d4e03570c.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/libinput.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/libinput.c b/src/libinput.c
index 3cc4426..595b177 100644
--- a/src/libinput.c
+++ b/src/libinput.c
@@ -2242,9 +2242,6 @@ LibinputInitScrollMethodsProperty(DeviceIntPtr dev,
		return;

	scroll_methods = libinput_device_config_scroll_get_default_method(device);
-	if (scroll_methods == LIBINPUT_CONFIG_SCROLL_NO_SCROLL)
-		return;
-
	if (scroll_methods & LIBINPUT_CONFIG_SCROLL_2FG)
		methods[0] = TRUE;
	if (scroll_methods & LIBINPUT_CONFIG_SCROLL_EDGE)
--
2.4.1
